.github/workflows/tests.yml

```diff
@@ -0,0 +1,31 @@
+name: prometheus_eaton_ups_exporter tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.11', '3.12']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: git config
+      run: |
+        git config --global user.email "eaton@exporter.com"
+        git config --global user.name "Eaton Exporter"
+    - name: Install dependencies
+      run: |
+        pip install . flake8
```

Comment:
IIRC your previous PR included Pyre. Any reason to drop it?

Reply:
Yes, that it is complaining a lot about `Undefined import [21]: Could not find a module corresponding to import "vcr".` and many others.

And while I plan to look into this and fix it, I did not want that to be a blocker for the rest of the uplifting which IMO provides a nice ground for such fixes/addons.
Note: Another thing I want to check too is the pytests, which fail for me locally, and might be possible to add to the CI, too (at least in so far these can be run without and actual response).

Reply:
That happens for modules that don't have typing available. `ruamel` has the same issue.

You can pyre-ignore the import:
```
import vcr  # pyre-ignore[21] 
```

Reply:
Note: Another thing I want to check too is the pytests, which fail for me locally, and might be possible to add to the CI, too (at least in so far these can be run without and actual response).

Ahh, that is indeed a problem. ;-)